web_widget_image_url/static/src/js/web_widget_image_url.js

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 - 2018 Modoolar <info@modoolar.com>
+// License LGPLv3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html).
+
+odoo.define('web_widget_image_url.FieldImageURL', function (require) {
+    "use strict";
+
+    var AbstractField = require('web.AbstractField');
+    var core = require('web.core');
+    var registry = require('web.field_registry');
+    var _t = core._t;
+
+    var UrlImage = AbstractField.extend({
+        className: 'o_attachment_image',
+        template: 'FieldImageURL',
+        placeholder: "/web/static/src/img/placeholder.png",
+        supportedFieldTypes: ['char'],
+
+        url() {
```

Comment:
change this to `url: function () {`

Reply:
@tarteo 
Fixed all javascript-lint related error.